lib/mumuki/domain/incognito.rb

```diff
@@ -83,6 +83,20 @@ def has_accepted?(term)
       false
     end
 
+    # ========
+    # Preferences
+    # ========
+
+    def preferences
+      Class.new do
```

Comment:
:warning: This is creating a new class in each method call, but you could use always the same class and even the same instance.

Reply:
Right. Do you mean like this? Or is there a better option?

```suggestion
      @preferences ||= Class.new do
```

Reply:
I was actually thinking of a named module like `module NullPreferences` within incognito